ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -320,5 +320,19 @@
     "CaseEventID": "addStatementOfService",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
+  },
+  {
```

Comment:
We can't merge this change alone, as James' changes need to go in at the same time, unless we merge this without permissions?